### PR TITLE
add _TZE200_mby4kbtq

### PR DESCRIPTION
**Pull request by b2un0**
relates to #Koenkk/zigbee2mqtt/issues/17684

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -1002,6 +1002,33 @@ const definitions: Definition[] = [
...
+            e.gas(),
+            tuya.exposes.gasValue().withUnit('LEL'),
+            e.binary('preheat', ea.STATE, true, false).withDescription('Indicates sensor preheat is active'),
+            tuya.exposes.faultAlarm(),
+            e.binary('alarm_switch', ea.STATE_SET, true, false),
+            tuya.exposes.silence(),
+        ],
+        meta: {
+            tuyaDatapoints: [
+                [1, 'gas', tuya.valueConverter.trueFalse0],
```
Comment (Koenkk) [inline src/devices/tuya.ts:1023]: Data points are similar to `TS0601_gas_sensor_2`, can you see if this device also supports the additional data points that `TS0601_gas_sensor_2` defines (e.g. `[7, 'alarm_time', tuya.valueConverter.raw],`)

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/2706

File: src/devices/tuya.ts (hunk 1 above)
Comment (b2un0) [reply src/devices/tuya.ts:1023]: However, if the "datapoints" match those of "TS0601_gas_sensor_2", the "alarm_switch" on DP 13 would be missing (is not activated in "TS0601_gas_sensor_2").

Comment (b2un0) [reply src/devices/tuya.ts:1023]: these are the datapoints from the tuya dev console
https://github.com/b2un0/z2m-device-converter/blob/main/tuya/_TZE200_mby4kbtq.json5

Comment (Koenkk) [reply src/devices/tuya.ts:1023]: Then let's leave it as is! Thanks!

**Pull request merged**